### PR TITLE
Update boto3 and issue/PR stats for October

### PR DESCRIPTION
**Pull request by willthames**

### CODE REVIEW

File: group-aws/boto3.md [hunk 1]
```diff
@@ -23,23 +23,25 @@ enforce that new features must have tests.
...
+# State of the codebase (As at 2017-10-08)
 
 ## boto3 only
 
 * `_ec2_vpc_dhcp_options_facts`
 * `_iam_cert_facts`
 * `_s3`
+* `aws_acm_facts`
 * `aws_api_gateway`
-* `aws_config`
```
Comment (willthames) [inline group-aws/boto3.md]: shouldn't have been here in the first place

File: group-aws/boto3.md [hunk 2]
```diff
@@ -100,7 +104,6 @@ TODO: add many more to the above list!
 * `lambda_facts`
 * `lambda_policy`
 * `lightsail`
-* `rds_facts`
```
Comment (willthames) [inline group-aws/boto3.md]: shouldn't have been here in the first place

File: group-aws/boto3.md [hunk 3]
```diff
@@ -123,17 +126,16 @@ TODO: add many more to the above list!
 
 ## boto only
 
-* `_ec2_elb`
-* `_ec2_elb_facts`
-* `_ec2_elb_lb`
 * `_ec2_remote_facts`
 * `_ec2_vpc`
 * `_ec2_vpc_dhcp_options`
-* `_rds`
```
Comment (willthames) [inline group-aws/boto3.md]: shouldn't have been here in the first place

File: group-aws/review.md [hunk 4]
```diff
@@ -13,8 +13,9 @@ the number of issues and PRs.
 |------------|-------------|----------|------------|---------|
 | 2017-07-05 | 128         | 161      | +32/-27    | +61/-68 |
 | 2017-09-05 | 144         | 131      | +11/-27    | +20/-78 |
+| 2017-10-08 | 334         | 147      | +196/-54   | +40/-76 |
```
Comment (s-hertel) [inline group-aws/review.md]: Keeping track of this was a great idea.

**Pull request merged**